test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,41 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+
+    @Test
+    public void isAnyNull() {
+        // no null
+        assertNoNull("123", 12);
+        assertNoNull("");
+
+        // only one null
+        assertHasNull("1232", null, 1009);
+        assertHasNull(10, null);
+        assertHasNull(null, 1223);
+
+        // multiple nulls
+        assertHasNull(20.0, null, null);
+        assertHasNull("Joks", null, null, null, null);
+        assertHasNull(null, null, null, "pop");
+
+
+    }
+
+    private void assertNoNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
+
+    private void assertHasNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
```

Comment:
Please use a clean PR for LO submissions as it would greatly improve the readability of your changes.